src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -396,7 +396,7 @@ def wait_for_create_job_to_complete(
     )
 
 
-def _hash_attachments(
+def hash_attachments(
```

Comment:
That makes sense, inside the function `hash_attachments` does call `hash_assets_and_create_manifest`, which is the actual job attachments functionality. While it doesn't directly call boto3, it does act like an API which calls the actual hashing functionality. It also works with callback info, which belongs in client ? I think it can stay in client api, but maybe under a new file ?

Reply:
will add to future refactoring

Reply:
Refactoring issues were discussed to be put in another commit to be addressed, as there are quite a few things that can be refactored with existing job attachments code, and don't really align with the snapshot functionality.